hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -2127,6 +2160,11 @@ func reconcileControlPlaneOperatorDeployment(deployment *appsv1.Deployment, hc *
 		})
 	}
 
+	if hc.Spec.AdditionalTrustBundle != nil {
```

Comment:
Ah good point, I think I did it like this thinking it would be good to move `setDeploymentTrustBundle` into a common location so we can de-duplicate adding the cert volume/mount code added in other files - what do you think about that, maybe I should refactor so the code moves e.g to [support/util/volumes](https://github.com/openshift/hypershift/blob/main/support/util/volumes.go) ?

Reply:
sgtm, thx!

Reply:
Now done